src/Microsoft.AspNetCore.Blazor.Razor.Extensions/Resources.resx

```diff
@@ -135,6 +135,9 @@
   <data name="ChildContentParameterName_Documentation" xml:space="preserve">
     <value>Specifies the parameter name for the '{0}' lambda expression.</value>
   </data>
+  <data name="ChildContentParameterName_TopLevelDocumentation" xml:space="preserve">
+    <value>Specifies the parameter name for all child content lambda expressions.</value>
```

Comment:
Calling them "lambda expressions" here is interesting - it implies the user has some understanding of the shape of the generated code. TBH I'm fine with it, but I wondered what you thought about calling them "child content blocks" or something instead (also on L136). Please don't feel pressured to change this unless you really want!

Reply:
yeah I thought that as well when I wrote it. I'll change this.